src/controllers/bookings.controller.js

```diff
@@ -1,6 +1,6 @@
 import BookingService from '../services/booking.service';
 import UserService from '../services/user.service';
-import sendResult from '../utils/sendResult';
+import sendResult from '../utils/result.util';
```

Comment:
great job @urbanishimwe  but this should be PascalCase

Reply:
isn't it pascal case??

Reply:
that's camelCase

Reply:
Okay first test your routes, we will get back to that routes